pkg/tracker/timeOut.go

```diff
@@ -81,14 +88,14 @@ func (b *TimeOutTracker) Exec(ctx context.Context) error {
 		address := "000000000000000000000000" + addr[2:]
 		decoded, err := hex.DecodeString(address)
 		if err != nil {
-			log.Fatal(err)
+			return errors.Wrapf(err, "decoding address")
 		}
 		hash := solsha3.SoliditySHA3(decoded)
 		var data [32]byte
 		copy(data[:], hash)
 		status, err := instance.GetUintVar(nil, data)
 		if err != nil {
-			fmt.Printf("Could not get staker timeOut status for miner address %s: %v\n", addr, err)
+			level.Error(b.logger).Log("msg", "getting staker timeOut status for miner", "address", addr, "err", err)
```

Comment:
I am not sure about this one. Why would it continue the execution when there is an error?
This means that it will save `0` in the DB. @themandalore can probably give some context here.